runtime/parachains/src/inclusion_inherent.rs

```diff
@@ -39,6 +39,8 @@ use crate::{
 };
 use inherents::{InherentIdentifier, InherentData, MakeFatalError, ProvideInherent};
 
+const BLOCK_WEIGHT_MARGIN: u64 = 1000;
```

Comment:
This value is entirely arbitrary; suggestions for better values would be welcomed.

Reply:
@coriolinus can you work with some of the folks more familiar with the benchmarking / weights in practice on this?

Reply:
Polkadot adjust fee with `SlowAdjustingFeeUpdate`, thus fees slowly decrease or increase depending on if the weight of the normal-class extrinsics of the block fill more or less than 25% of what is available to them (See `TargetBlockFullness`. So this can be a first indicator.

Then this value is configured when defining the runtime. inside pallets I guess best is to make it configurable in `Config` or to `system::Config::BlockWeights::get().max_block` maybe.

Reply:
maybe this inherent could be simply benchmarked, then we would just check that block weight can handle this inherent weight.